src/js/chat/templates.js

```diff
@@ -42,7 +42,7 @@ var linkify = exports.linkify = function(message) {
 };
 
 var escapeEmoteCode = function(code) {
-    return code.replace(/('|"|>|<|&)/g, '');
```

Comment:
By encode I mean html entity encoding.. `<` becomes `&lt;`, `>` becomes `&gt;` and `&` becomes `&amp;`